app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -0,0 +1,51 @@
+require 'json'
+
+module Idv
+  module InPerson
+    class UspsLocationsController < ApplicationController
+      include IdvSession
+      include UspsInPersonProofing
+
+      # get the list of all pilot Post Office locations
+      def index
+        usps_response = []
+        begin
+          usps_response = Proofer.new.request_pilot_facilities
+        rescue Faraday::ConnectionFailed => _error
+          nil
+        end
+
+        render json: usps_response.to_json
+      end
+
+      # save the Post Office location the user selected to the session
+      def update
+        idv_session.applicant ||= {}
+        idv_session.applicant[:selected_location_details] = permitted_params.as_json
+
+        render json: { success: true }, status: :ok
+      end
+
+      # return the Post Office location the user selected from the session
+      def show
+        selected_location = idv_session.applicant&.[](:selected_location_details)&.to_json || {}
+
+        render json: selected_location, status: :ok
+      end
+
+      protected
+
+      def permitted_params
+        params.require(:usps_location).permit(
+          :addressLine2,
+          :name,
+          :phone,
+          :saturdayHours,
+          :streetAddress,
+          :sundayHours,
+          :weekdayHours,
+        )
```

Comment:
I feel like we snake_case all params from the client, what's the level of effort to use snake_case over the wire here?

Reply:
This will be addressed with LG-7025

Reply:
I decided to address this in this PR and did so in 34a7cc50594937453792c6791e0b108e5575c504